pysat/tests/test_utils.py

```diff
@@ -36,8 +35,7 @@ def remove_files(inst):
     # remove any files
     dir = inst.files.data_path
     for the_file in os.listdir(dir):
-        if (the_file[0:13] == 'pysat_testing') & (the_file[-19:] ==
-                                                  '.pysat_testing_file'):
+        if (the_file == 'test_ncdf.nc'):
```

Comment:
I made the name purposely long and specific to pysat to ensure we don't ever accidentally delete a user's file

Reply:
Will update the test file name

Reply:
fixed.  I think this was originally a copy and paste error from the test_files routine, where the pysat_testing?????????.pysat_testing_file routine is used.  Here the files created are just 'test_ncdf.nc'.  Updated the file name here to 'pysat_test_ncdf.nc', which should be unique.